src/main/webapp/app/overview/course-card.scss

```diff
@@ -1,13 +1,41 @@
 .card {
     min-height: 320px;
 
+    @keyframes on-hover {
+        0% {
+            transform: scale(1);
+        }
+
+        33% {
+            transform: scale(1.012);
+        }
+
+        100% {
+            transform: scale(1.012);
+        }
+    }
+
+    &:hover {
+        animation: on-hover 0.5s linear;
+        animation-fill-mode: forwards;
+        background-color: var(--hover-slightly-darker-body-bg);
+    }
```

Comment:
Honestly, I don't think I get the idea here.

You set the animation for 0.5s but in the animation only animate for 33% of that time.

Why not just set the keyframes 0% and 100% and a length of 166ms? I would even give it 2s actually.